framework/src/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -564,9 +566,32 @@ protected void addUncheckedCodeDefaults(QualifierDefaults defs) {
                         AnnotationUtils.fromClass(elements, annotation), TypeUseLocation.OTHERWISE);
             }
         }
+    }
+
+    /**
+     * Adds standard unchecked defaults that do not conflict with previously added defaults.
+     *
+     * @param defs {@link QualifierDefaults} object to which defaults are added
+     */
+    protected void addUncheckedStandardDefaults(QualifierDefaults defs) {
         Set<? extends AnnotationMirror> tops = this.qualHierarchy.getTopAnnotations();
         Set<? extends AnnotationMirror> bottoms = this.qualHierarchy.getBottomAnnotations();
         defs.addUncheckedStandardDefaults(tops, bottoms);
+    }
+
+    /**
+     * Ensure that we have defaults specified.
```

Comment:
Do use "we" in comments.  Instead, you could just say "Ensure defaults are specified".  But that isn't what the method is actually doing.  It's checking that a default qualifier in (at least one) hierarchy has been set and issue an error if it has not.